static/includes/SMBVFSConfigurationWarning.md

```diff
@@ -0,0 +1,32 @@
+&NewLine;
+
+{{< hint type=info title="Manual VFS Configuration Is Not Supported" >}}
+Many online guides recommend manually configuring SMB VFS options for macOS compatibility:
+
+```
+vfs objects = catia fruit streams_xattr
+```
+
+TrueNAS does not support manual VFS module configuration. Manually setting VFS objects via Auxiliary Parameters or CLI overrides TrueNAS defaults and breaks Asynchronous I/O (AIO), Access Control Lists (ACLs), and Shadow Copies.
```

Comment:
If Auxiiliary Parameters is still a UI setting it should be in bold, otherwise make it lower case.

```suggestion
TrueNAS does not support manual VFS module configuration. Manually setting VFS objects via **Auxiliary Parameters** or CLI overrides TrueNAS defaults and breaks Asynchronous I/O (AIO), Access Control Lists (ACLs), and Shadow Copies.
```
```suggestion
TrueNAS does not support manual VFS module configuration. Manually setting VFS objects via auxiliary parameters or CLI overrides TrueNAS defaults and breaks Asynchronous I/O (AIO), Access Control Lists (ACLs), and Shadow Copies.
```